.github/workflows/ci.yml

```
@@ -140,22 +140,16 @@ jobs:
         with:
           python-version: "3.10"
 
-      - run: python -m pip install tox
+      #- run: python -m pip install tox
 
         # Get only the latest tagged version for stability reasons
       - name: Install virtualenv
-        run: git clone https://github.com/pypa/virtualenv.git
+        #run: git clone https://github.com/pypa/virtualenv.git
+        run: pip install virtualenv
         shell: bash
 
       - name: Test Nushell in virtualenv
-        run: |
-          cd virtualenv
-          # if we encounter problems with bleeding edge tests pin to the latest tag
-          # git checkout $(git describe --tags | cut -d - -f 1)
-          # We need to disable failing on coverage levels.
-          nu -c "open pyproject.toml | upsert tool.coverage.report.fail_under 1 | save patchproject.toml"
-          mv patchproject.toml pyproject.toml
-          tox -e ${{ matrix.py }} -- -k nushell
+        run: cd scripts; nu test_virtualenv.nu
```

can we just do
```suggestion
        run: nu scripts/test_virtualenv.nu
```
here?